config/locales/en/course/mailer/user_invitation_email.yml

```diff
@@ -7,5 +7,7 @@ en:
           You are invited to register for %{course} on %{coursemology}.
 
 
-          You do not seem to have an account registered at this email address; please %{sign_up}
-          for an account, and register for %{course} using the following registration code:
+          You do not seem to have an account registered at this email address (%{email});
+          please %{sign_up} for an account before visiting %{course} to enrol for the course.
+          If you have an existing Coursemology account registered under a different email,
+          enter the following personal registration code when you enrol:
```

Comment:
I put 'enter' to make the action explicit. Obvious, but I think it helps the user think less by invoking an image ('key in' might be even better). Do you think it helps?

Would "non-transferable" instead of "personal" be better?